include/effects/strip/fireeffect.h

```diff
@@ -226,14 +229,16 @@ class PaletteFlameEffect : public FireEffect
                        bool reversed = false,
                        bool mirrored = false)
         : FireEffect(strName, ledCount, cellsPerLED, cooling, sparking, sparks, sparkHeight, reversed, mirrored),
-          _palette(palette)
+          _palette(palette), 
+          _ignoreGlobalColor(ignoreGlobalColor)
     {
         construct();
     }
 
     PaletteFlameEffect(const JsonObjectConst& jsonObject)
       : FireEffect(jsonObject),
-        _palette(jsonObject[PTY_PALETTE].as<CRGBPalette16>())
+        _palette(jsonObject[PTY_PALETTE].as<CRGBPalette16>()),
+        _ignoreGlobalColor(jsonObject["igc"])

```

Comment:
Now that "igc" is used in multiple effects with a defined meaning, I think it's best to add a definition (`PTY_IGNOREGLOBALCOLOR` seems sensible) to effects.h for it, and use that in all 3 effects that now use "igc".

Reply:
Re: PTY_IGNOREGLOBALCOLOR not a bad idea. I will make the change and push a new commit.